docs/contributing-to-lagoon/contributing.md

```diff
@@ -1,10 +1,10 @@
 # Contributing
 
-We gladly welcome any and all contributions to Lagoon! 
+We gladly welcome any and all contributions to Lagoon!
 
-### What kind of contributions do we need? Examples!
```

Comment:
This is a stupid rule, I want that exclamation point there and I'm going to put it back.

Reply:
I think the linting I applied removes ending punctuation for titles, I can revert those